typescript/cli/src/send/transfer.ts

```diff
@@ -40,51 +38,41 @@ export async function sendTestTransfer({
 }: {
   context: WriteCommandContext;
   warpCoreConfig: WarpCoreConfig;
-  origin?: ChainName; // resolved in signerMiddleware
-  destination?: ChainName; // resolved in signerMiddleware
+  chains: ChainName[];
   amount: string;
   recipient?: string;
   timeoutSec: number;
   skipWaitForDelivery: boolean;
   selfRelay?: boolean;
 }) {
-  const { chainMetadata } = context;
-
-  if (!origin) {
```

Comment:
Deleted because this does not really work when an origin/destination is provided, but not part of the WarpCoreConfig. Also, can just use `--origin` and `--destination`.